addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2552,6 +2554,7 @@ const Wysiwyg = Widget.extend({
     },
     _stopPeerToPeer: function () {
         this.ptp && this.ptp.stop();
+        this._onHistoryResetFromSteps = () => {};
```

Comment:
Wait, those `_onHistoryResetFromSteps` changes actually have an effect on the unpacking ? I'm confused.